make.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+set -e
+
+# Script inspired by https://gist.github.com/szeidner/613fe4652fc86f083cefa21879d5522b
+
+PROGNAME=$(basename $0)
+WORKING_DIR=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+
+die() {
+    echo "$PROGNAME: $*" >&2
+    exit 1
+}
+
+usage() {
+    if [ "$*" != "" ] ; then
+        echo "Error: $*"
+    fi
+
+    cat << EOF
+Usage: $PROGNAME --bundle-id [BUNDLE_ID_PRODUCTION] --bundle-id-staging [BUNDLE_ID_STAGING] --app-name [PROJECT_NAME]
+Set up an iOS app from tuist template.
+Options:
+-h, --help                         display this usage message and exit
+-b, --bundle-id [BUNDLE_ID_PRODUCTION]  new production id (i.e. com.example.package)
+-s, --bundle-id-staging [BUNDLE_ID_STAGING]  new staging id (i.e. com.example.package.staging)
+-n, --app-name [PROJECT_NAME]          new app name (i.e. MyApp, "My App")
+EOF
+
+    exit 1
+}
+
+bundleidproduction=""
+bundleidstaging=""
+appname=""
+while [ $# -gt 0 ] ; do
+    case "$1" in
+    -h|--help)
+        usage
+        ;;
+    -b|--bundle-id)
+        bundleidproduction="$2"
+        shift
+        ;;
+    -s|--bundle-id-staging)
+        bundleidstaging="$2"
+        shift
+        ;;
+    -n|--app-name)
+        appname="$2"
+        shift
+        ;;
+    -*)
+        usage "Unknown option '$1'"
+        ;;
+    *)
+        usage "Too many arguments"
+      ;;
+    esac
+    shift
+done
+
+OLD_APPNAME="ProjectName"
+OLD_BUNDLE_PRODUCTION="co.nimblehq.projectname"
+OLD_BUNDLE_STAGING="co.nimblehq.projectname.staging"
```

Comment:
I prefer using only `{PROJECT_NAME}`, `{BUNDLE_ID_PRODUCTION}`, `{BUNDLE_ID_STAGING}` to prevent people to use these old strings in other places.
I think we can rename all these old strings to constant values.
For example:
<img width="319" alt="Screen Shot 2021-10-05 at 18 16 54" src="https://user-images.githubusercontent.com/19943832/136013083-197be5bf-784b-4856-92b5-93694a8b7685.png">

What do you think?